defaults/main.yml

```diff
@@ -204,6 +205,7 @@ openio_oioswift_filter_container_hierarchy:
   sentinel_name: "{{ openio_oioswift_namespace }}-master-1"
   redis_keys_format: v2
```

Comment:
You can remove `redis_keys_format: v2` for oioswiftext.
v2 is no more supported, please update to v3